internal/hcs/schema2/registry_value_type.go

```diff
@@ -0,0 +1,17 @@
+package hcsschema
+
+// NOTE: manually added
+
+type RegistryValueType string
+
+// List of RegistryValueType
+const (
```

Comment:
The Registry value types are also defined in `internal/winapi` (as their corresponding int values though), should we keep this there so that everything is together?

Reply:
Since `winapi` uses the Win32 API enum int values, and HCS wants its own string consts, we probably should keep them separate.
I don't think we want to maintain a lookup between the string and int values.